rasa/core/actions/two_stage_fallback.py

```diff
@@ -194,7 +194,7 @@ def _second_affirmation_failed(tracker: DialogueStateTracker) -> bool:
 
 
 def _message_clarification(tracker: DialogueStateTracker) -> List[Event]:
-    clarification = copy.deepcopy(tracker.latest_message)
+    clarification = copy.deepcopy(cast(Event, tracker.latest_message))
```

Comment:
I've added the type hint on the tracker, but there still an issue to fix to remove the casting: in case `latest_message` is empty, what should be done? Raise? Return `[ActionExecuted(ACTION_LISTEN_NAME)]`?

Reply:
How about raising a `TypeError` in case it's `null`? There should always be a `latest_message` != None